### ISSUE TITLE
muted optie bij opstart

### ISSUE THREAD

**Issue by nobels** [MakeInBelgium/babbelbox#53]
Als een leerkracht een groep van 18 mensen zou uitnodigen.
Dan zie ik toch een paar problemen:
-> echo effect omdat er mensen met of zonder headset werken.
-> veel mensen die tegelijk praten
-> te veel noise

De oplossing of optie is er maar wil dit toch even voor stellen:
> audioMuteStatusChanged - event notifications about audio mute status changes. The listener will receive an object with the following structure:
> 
> {
>     muted: boolean // new muted status - boolean
> }

Ik had gedacht aan advanced opties als collapsable div, en het daar bij te zetten.
je kan daar ook de kwaliteit ook op het laagste zetten daar. (extra optie)

**Comment by toonvandeputte** [comment]
Klopt, maar ik denk dat er voor dit scenario betere oplossingen zijn dan Praatbox. Smartschool Live schijnt goed te werken, en ook Zoom, Whereby, etc. worden vlot gebruikt. Ik heb ook al opgevangen dat zelfs Instagram en Facebook Live al gebruikt zijn voor dat soort dingen. Omdat leerlingen vaak sowieso al toegang hebben tot die platformen.
